boards/arm/mimxrt1160_evk/mimxrt1160_evk.dtsi

```diff
@@ -33,6 +34,26 @@
 			pwms = <&flexpwm1_pwm2 0>;
 		};
 	};
+
+
+	power-states {
+
+		idle: set_point_1_wait {
+			/* idle corresponds to set point 1 (wait) for RT1170 */
+			compatible = "zephyr,power-state";
+			power-state-name="runtime-idle";
+			substate-id = <IMX_SPC_SET_POINT_1_WAIT>;
+			min-residency-us = <10>;
+		};
+
+		suspend: set_point_10_stop {
```

Comment:
s/set_point_10_stop/set_point_10_suspend ?

Reply:
Good point. Renamed it.